src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksTaxRuleDependencyServiceImpl.java

```diff
@@ -121,7 +121,7 @@ protected void poulateForeignVendorValues(PaymentWorksVendor pmwVendor, KfsVendo
         }
     }
     
-    protected void populateOwernshipCode(PaymentWorksVendor pmwVendor, TaxRule taxRule, VendorHeader vendorHeader) {
+    protected void populateOwnernshipCode(PaymentWorksVendor pmwVendor, TaxRule taxRule, VendorHeader vendorHeader) {
```

Comment:
There's still a typo in this method name.

Reply:
thanks, fixed